dask/array/core.py

```diff
@@ -536,7 +535,9 @@ def blockdims_from_blockshape(shape, blockshape):
     ((4, 4, 2), (3, 3, 3, 1))
     """
     if blockshape is None:
```

Comment:
`normalize_chunks` uses this function.  I do still like these names when we need to distinguish between the two concepts.